src/main/java/eu/europeana/metis/sandbox/config/SandboxConfig.java

```diff
@@ -55,10 +55,13 @@
   @Value("${sandbox.enrichment.enrichment-url}")
   private String enrichmentServiceUrl;
 
-  private String defaultXsltUrl;
-
+  //TODO: 04-03-2021 We should remove this configuration once
+  //TODO: XsltTransformation in metis is fixed
+  @Value("${sandbox.url.edm-sorted-url}")
   private String edmSorterUrl;
```

Comment:
I think now the method `edmSorterUrl()` is no longer used.

Reply:
Also, the file referenced in that method (`classpath:edm/edm.xsd.sorter.xsl`) can be removed, I think?